roles/ceph-common/tasks/main.yml

```diff
@@ -1,55 +1,33 @@
 ---
-- name: include installs/install_on_redhat.yml
-  include: installs/install_on_redhat.yml
-  when:
-    - ansible_os_family == 'RedHat'
+- include_tasks: installs/install_on_redhat.yml
```

Comment:
@rishabh-d-dave can you check and keep this syntax if possible ?
```
- name: include_tasks installs/install_on_redhat.yml
  include_tasks: installs/install_on_redhat.yml
```

Reply:
@rishabh-d-dave I was concerned because of the output in the playbook but I've just made some tests, not sure it's worth to keep this syntax.

Reply:
For most tasks, the description in the `name` field looked redundant. So, IMO it's okay to let them go for now.

Reply:
@leseb that was my concern

Reply:
@leseb 
Yes. I'll modify the patch accordingly.